frontend/components/events/Guests.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { View, ScrollView, Text, Pressable, Button, StyleSheet } from 'react-native';
+import { ListItem, Avatar } from '@rneui/themed';
+import userData from '../home/exampleData/userData.js';
+
+const Guests = ({setOpen, DYNAMICUSERINFO}) => {
+  const sampleData = userData;
+  // console.log('SAMPLE DATA: ', sampleData);
+  const [invitees, setInvitees] = useState([]);
+
+  const handleInvite = (guestId) => {
+    const userIdInList = invitees.indexOf(guestId);
+    if (userIdInList !== -1) {
+      invitees.splice(userIdInList, 1);
+    } else {
+      invitees.push(guestId);
+      setInvitees([...invitees]);
+      console.log(invitees);
```

Comment:
Are the invitees going to be logged when clicking on any of the dogs listed in guests? I tried clicking on them but nothing logs. Let me know if it works differently.

Reply:
Make sure to pull the latest change from main before merging this PR

Reply:
Just tested. It works, but the area which triggers the button is not the entire width of the screen -- it's just a small area surrounding the text/avatar. Haven't applied styling to the cards yet. Will pull changes and re-push!